balboa-client-dispatcher/src/main/scala/com/socrata/balboa/config/DispatcherConfig.scala

```diff
@@ -0,0 +1,33 @@
+package com.socrata.balboa.config
+
+import com.socrata.balboa.config.ClientType.ClientType
+import com.socrata.balboa.metrics.config.{Configuration, Keys}
+import org.slf4j.LoggerFactory
+
+/**
+ * All inclusive Configuration file
+ */
+object DispatcherConfig {
+
+  private val Log = LoggerFactory.getLogger("DispatcherConfig")
+
+  /**
+   * @return Client Types parsed from configuration.
+   */
+  def clientTypes: Array[ClientType] = {
+    val clientStrings: Array[String] = Configuration.get().getString(Keys.DISPATCHER_CLIENT_TYPES).split(',')
```

Comment:
The actual Configuration class we made is not a direct subclass or relates to the Apache Configuration that is referenced here.  Configuration is actually a direct subclass of http://docs.oracle.com/javase/7/docs/api/java/util/Properties.html.  It might be worth utilizing the common Apache Configuration class one day but for now lets keep it the way it is.